README.md

```diff
@@ -111,24 +112,36 @@ module.exports = {
             }
         },
         android: {
-            version: 1.0.1,
-            thirdparty: {}
+            vendor: {}
         },
         windows: {
-            version: 1.0.0,
-            thirdparty: {}
+            vendor: {}
         }
     }
 
 ```
 
+And this is what the `package.json` could look like:
+
+```
+{
+    "name": "ti-mymodule",
```

Comment:
Shouldn't you also standardise on the prefix used in the module? This will make the hyperloop modules distinguishable in the NPM ecosystem.